account_receipt_print/i18n/it.po

```diff
@@ -0,0 +1,178 @@
+msgstr "<strong>Data ricevuta:</strong>"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "<strong>Reference:</strong>"
+msgstr "<strong>Riferimento:</strong>"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "<strong>Source:</strong>"
+msgstr "<strong>Origine:</strong>"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "<strong>Subtotal</strong>"
+msgstr "<strong>Imponibile</strong>"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "<strong>Total</strong>"
+msgstr "<strong>Totale</strong>"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Cancelled Receipt"
+msgstr "Ricevuta annullata"
+
+#. module: account_receipt_print
+#: model:ir.model.fields,field_description:account_receipt_print.field_account_move__display_name
+msgid "Display Name"
+msgstr "Nome visualizzato"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Draft Receipt"
+msgstr "Ricevuta in bozza"
+
+#. module: account_receipt_print
+#: model:ir.model.fields,field_description:account_receipt_print.field_account_move__id
+msgid "ID"
+msgstr "ID"
+
+#. module: account_receipt_print
+#: model:ir.model,name:account_receipt_print.model_account_move
+msgid "Journal Entry"
+msgstr "Registrazione contabile"
+
+#. module: account_receipt_print
+#: model:ir.model.fields,field_description:account_receipt_print.field_account_move____last_update
+msgid "Last Modified on"
+msgstr "Ultima modifica il"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Please use the following communication for your payment:"
+msgstr "Per il pagamento utilizzare la seguente indicazione:"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.view_move_form_receipt
+msgid "Print"
+msgstr "Stampa"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Purchase Receipt"
+msgstr "Ricevuta di acquisto"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Receipt"
+msgstr "Ricevuta"
+
+#. module: account_receipt_print
+#: model:ir.actions.report,name:account_receipt_print.account_receipts
+msgid "Receipts"
+msgstr "Ricevute"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "Tax ID"
+msgstr "Partita IVA"
+
+#. module: account_receipt_print
+#: model_terms:ir.ui.view,arch_db:account_receipt_print.report_receipt_document
+msgid "on"
+msgstr "su"
+
```

Comment:
Are the following lines needed? Otherwise, could you please remove them?

Reply:
Done!